src/components/Home/Hero.tsx

```diff
@@ -1,16 +1,26 @@
+import { heroVideoBackgroundIphone } from "@/constant";
 import { useLoadSectionRefAndId } from "@/hooks/helpers/useLoadSectionRefAndId";
 import { LazyMotion, m } from "framer-motion";
 import NextLink from "next/link";
 import DownArrow from "public/assets/down-arrow.svg";
 import OOLogo from "public/assets/oo-logo.svg";
-import { useRef } from "react";
+import { useEffect, useRef } from "react";
 
 const loadFeatures = () => import("../../utils/features").then((res) => res.default);
 
 export default function Hero() {
   const ref = useRef<HTMLDivElement>(null);
   useLoadSectionRefAndId(ref, "");
 
+  useEffect(() => {
+    if (typeof window === "undefined") return;
```

Comment:
the video renders with a slightly different color on iphones. my best guest is this is to do with the video codec. this meant that if we match the color to the background on iphones, then it won't match on other devices.

the solution here is to use the `navigator.platform` property (technically this is supposed to be deprecated but after looking into the conversation around it, it looks like its still going to be around for a long time.

if we detect that we are on an ios device, we update the value of the global css variable that we use for the hero video background color.